CRM/Core/Payment/Stripe.php

```diff
@@ -352,11 +352,11 @@ function doDirectPayment(&$params) {
     */
 
     // drastik - Uncomment this for Drupal debugging to dblog.
-   /* 
+    
```

Comment:
We can't commit this un-commented, because this is Drupal-specific (some people are on Wordpress / Joomla).